packages/dashboard/src/components/FooterLine/links.json

```diff
@@ -0,0 +1,9 @@
+{
+    "MASK_OFFICIAL_WEBSITE": "https://mask.io",
```

Comment:
Maybe this should be moved to constant package

Reply:
Constant package managed constants of web3. I'm not sure should move links to there.

Reply:
/cc @septs @guanbinrui

Reply:
i think no need.